appcues/build.gradle

```diff
@@ -12,9 +12,9 @@ apply from: 'codecov.gradle'
 apply from: 'dokka.gradle'
 
 ext.room_version = '2.5.2'
-ext.compose_version = '1.5.1'
-ext.compose_compiler_version = '1.5.3'
-ext.koin_version = '3.5.0'
+ext.compose_version = '1.5.2'
+ext.compose_compiler_version = '1.4.8'
+ext.koin_version = '3.4.3'
```

Comment:
note: Koin 3.5.0 depends on Kotlin 1.9 so that cannot be updated at this time either